### ISSUE TITLE
 Failed to start bean 'documentationPluginsBootstrapper' when starting rest version

### ISSUE THREAD

**Issue by carcuevas** [sbraconnier/jodconverter#315]
Hi, 

when trying to start the `jodconverter-sample-rest` using LibreOffice  version is:  `7.0.4-4` running in debian11 

```
soffice -accept=socket,host=127.0.0.1,port=2002,tcpNoDelay=1;urp;StarOffice.ServiceManager -headless -invisible -nocrashreport -nodefault -nofirststartwizard -nolockcheck -nologo -norestore -env:UserIn
stallation=file:///tmp/.jodconverter_socket_host-127.0.0.1_port-2002_tcpNoDelay-1
```

I am getting the following error:

```
2022-11-14 16:48:11.238 DEBUG 3732 --- [er-offprocmng-1] o.j.local.office.ExitCodeRetryable       : Execution attempt #1 failed, retrying after sleep of 250 ms
2022-11-14 16:48:11.284  INFO 3732 --- [ssageDispatcher] o.j.local.office.OfficeConnection        : Disconnected from 'socket,host=127.0.0.1,port=2003,tcpNoDelay=1'
2022-11-14 16:48:11.489 DEBUG 3732 --- [er-offprocmng-1] o.j.local.office.ExitCodeRetryable       : Execution attempt #2
2022-11-14 16:48:11.489 DEBUG 3732 --- [er-offprocmng-1] o.j.local.office.ExitCodeRetryable       : Execution succeeded on attempt #2
2022-11-14 16:48:11.489  INFO 3732 --- [er-offprocmng-1] o.j.l.office.LocalOfficeProcessManager   : Process exited with code 0
2022-11-14 16:48:11.489 DEBUG 3732 --- [er-offprocmng-1] o.jodconverter.core.office.OfficeUtils   : Deleting '/tmp/.jodconverter_socket_host-127.0.0.1_port-2003_tcpNoDelay-1'
2022-11-14 16:48:11.489 DEBUG 3732 --- [er-offprocmng-1] o.j.c.o.OfficeUtils$DeleteFileRetryable  : Execution attempt #1
2022-11-14 16:48:11.506 DEBUG 3732 --- [er-offprocmng-1] o.j.c.o.OfficeUtils$DeleteFileRetryable  : Execution succeeded on attempt #1
2022-11-14 16:48:11.506 DEBUG 3732 --- [           main] o.j.l.office.LocalOfficeProcessManager   : Stop task executed successfully.
2022-11-14 16:48:11.507 DEBUG 3732 --- [           main] o.jodconverter.core.office.OfficeUtils   : Deleting '/tmp/.jodconverter_16e7aea4-92b6-4f36-881b-6fcd6eb9a1cb'
2022-11-14 16:48:11.507 DEBUG 3732 --- [           main] o.j.c.o.OfficeUtils$DeleteFileRetryable  : Execution attempt #1
2022-11-14 16:48:11.507 DEBUG 3732 --- [           main] o.j.c.o.OfficeUtils$DeleteFileRetryable  : Execution succeeded on attempt #1
2022-11-14 16:48:11.521  INFO 3732 --- [           main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2022-11-14 16:48:11.553  INFO 3732 --- [           main] ConditionEvaluationReportLoggingListener :

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2022-11-14 16:48:11.601 ERROR 3732 --- [           main] o.s.boot.SpringApplication               : Application run failed

org.springframework.context.ApplicationContextException: Failed to start bean 'documentationPluginsBootstrapper'; nested exception is java.lang.NullPointerException: Cannot invoke "org.springframework.web.servlet.mvc.condition.PatternsRequest
Condition.getPatterns()" because "this.condition" is null
        at org.springframework.context.support.DefaultLifecycleProcessor.doStart(DefaultLifecycleProcessor.java:181) ~[spring-context-5.3.22.jar!/:5.3.22]
        at org.springframework.context.support.DefaultLifecycleProcessor.access$200(DefaultLifecycleProcessor.java:54) ~[spring-context-5.3.22.jar!/:5.3.22]
        at org.springframework.context.support.DefaultLifecycleProcessor$LifecycleGroup.start(DefaultLifecycleProcessor.java:356) ~[spring-context-5.3.22.jar!/:5.3.22]
```

Am I forgetting something here? 

Thanks

**Comment by EugenMayer** [comment]
I would mark this one as a dupe of https://github.com/sbraconnier/jodconverter/issues/317 since we have more progress in 317 (even though this issue was here first). You decision @sbraconnier

**Comment by EugenMayer** [comment]
@sbraconnier this one can also be closed since you merge #318 and closed #317

**Comment by sbraconnier** [comment]
Thanks @EugenMayer 

I'll try to release a version before Christmas
